loadtest/beats-yaml-generator.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+stream = os.popen('kubectl get pods -o wide -n logstash')
+output = stream.read()
+output = output.split("\n")
+result = ""
+
+i = 0
+for o in output:
+    if i > 0 and o != "":
```

Comment:
Instead of all this `I > 0`, `I >= 2` checking to print `,`, couldn't we just parse out all the hosts to a list and then join the list.